ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -259,10 +259,12 @@ def run_export(environment, dataset, task, action_name, expected_optimization_ty
 
     assert (
         exported_model.model_format == ModelFormat.OPENVINO
-    ), f"In action '{action_name}': Wrong model format after export"
+    ), f"In action '{action_name}': Wrong model format after export. " \
+       f"exported_model.model_format=exported_model.model_format"
```

Comment:
```suggestion
       f"{exported_model.model_format=}"
```
-- this is [f-strings with equal sign for debugging](https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging)